packages/mdc-list/README.md

```diff
@@ -135,6 +135,44 @@ OR
 </ul>
 ```
 
+### Single Selection List
+
+MDC List can handle selecting/deselecting list elements based on click or keyboard action. When enabled, the `space` and `enter` keys (or `click` event) will trigger an 
+single list item to become selected or deselected. 
+
+```html
+<ul id="my-list" class="mdc-list" aria-orientation="vertical">
+  <li class="mdc-list-item">Single-line item</li>
+  <li class="mdc-list-item">Single-line item</li>
+  <li class="mdc-list-item">Single-line item</li>
+</ul>
+```
+
+```js
+var listEle = document.getElementById('my-list');
+var list = new mdc.list.MDCList(listEle);
+list.singleSelection = true;
+```
+
+#### Pre-selected list item
+
+When rendering the list with a pre-selected list item, the list item that needs to be selected should contain
+the `mdc-list-item--selected` class before creating the list.
```

Comment:
Should the list item also have the `aria-selected="true"` attribute?

Reply:
No, it gets added automatically when the list component is created. We add the `--selected` class because we use that as our selector.

Reply:
Gotcha. Can you show me the line number where that occurs?

Reply:
https://github.com/material-components/material-components-web/blob/1d5f6106e02a3755ceb9eccaa2a0bf39444d08e8/packages/mdc-list/index.js#L106

Reply:
I don't see where `aria-selected="true"` gets added though. Could you point me to that?

Reply:
That's in the foundations `setSelectedIndex`.

Reply:
If the `selectedElement` is found it sets the `selectedIndex` to the `selectedElement` index. The `selectedIndex` proxies to the foundation `setSelectedIndex`, which sets the attributes on the list element. 

https://github.com/material-components/material-components-web/blob/cbbd3948ee793abddb5bdbdaa8208385f5e7addd/packages/mdc-list/index.js#L111

https://github.com/material-components/material-components-web/blob/cbbd3948ee793abddb5bdbdaa8208385f5e7addd/packages/mdc-list/index.js#L117

https://github.com/material-components/material-components-web/blob/cbbd3948ee793abddb5bdbdaa8208385f5e7addd/packages/mdc-list/foundation.js#L115

Reply:
Awesome, thanks for the patience and guiding me through. Since that means the `aria-selected` attribute is only set by the MDC component index.js implementation during initialization, let's add a note to include `aria-selected="true"` for the pre-selected list item. That will help users who wrap our components and our users in general.

Reply:
? `aria-selected` is set by the foundation

Reply:
If you wrap the component and want to use the single selection list, you have to call `setSelectedIndex` on the foundation so that it can initialize the initial selected value. `setSelectedIndex` also sets `aria-selected` on the list element that is selected.